src/main/res/values/strings.xml

```diff
@@ -96,12 +96,12 @@
     the applications it provides.</string>
   <string name="number_of_applications">Number of applications</string>
   <string name="ok">OK</string>
-  <string name="open_link_confirm_format">Open link %s?</string>
+  <string name="open_link_confirm_format">Open %s?</string>
```

Comment:
I argue here that people know what a link is by looking at it.

Reply:
I think, "Open %s?" is fine too.